_assets/scss/design-guide.scss

```diff
@@ -22,72 +22,135 @@ table {
   margin-bottom: $base-spacing * 2;
 }
 
-// Custom styling for the inverted feedback button in the header.
-// This will likely be removed/replaced by a global site search.
-// The @extend .inverted is not working in:
-// https://github.com/AusDTO/gov-au-ui-kit/blob/master/assets/sass/components/_page-header.scss#L288
 
-header[role='banner'] {
-  .page-header {
-    padding-top: 0;
-  }
+.hero {
+  padding-top: 0;
 
-  .page-header__logo {
-    padding-top: 0;
-    margin-bottom: $tiny-spacing;
+  .wrapper {
+    @include ie-lte(8) {
+      @include ie-clearfix;
+    }
+
+    background-image: url('/assets/hero.svg');
+    background-repeat: no-repeat;
+    background-position: calc(100% - 50%) calc(100% - 60%);
+    background-size: 70%;
+    //background-position: calc(100% - 15%) calc(100% - 25%);
 
     @include media($tablet) {
-      height: rem(132);
-      margin-bottom: 0;
+      background-position: bottom 10% left 8%;
+      background-size: 35%;
+      min-height: 0;
     }
 
-    .logo {
-      display: inline-block;
-      position: relative;
-      border-bottom: none;
+    @include media($desktop) {
+      background-position: bottom 0% left 8%;
+      background-size: 30%;
+      min-height: rem(350);
+    }
 
-      &:hover,
-      &:focus {
-        background-color: transparent;
+    .site-description {
+      margin-top: 55%;
+
+      @include media($tablet) {
+        width: 50%;
+        float: right;
+        margin-top: 2.5%;
+        margin-right: 5%;
       }
 
-      svg {
-        position: absolute;
-        top: 0;
-        width: rem(48);
-        height: auto;
+      @include media($desktop) {
+        width: 55%;
+        float: right;
+        margin-top: 5%;
+        margin-right: 5%;
+      }
+    }
+  }
+}
 
-        @include media($tablet) {
-          width: rem(64);
-        }
+.no-svg {
```

Comment:
Just a question but should be include the `no-svg` class as a default on our html tag? That way we have a fallback when js is off?